docs/pom.xml

```diff
@@ -48,6 +48,10 @@
                     <groupId>org.asciidoctor</groupId>
                     <artifactId>asciidoctor-maven-plugin</artifactId>
                     <configuration>
+                        <attributes>
+                            <javaee_version>8</javaee_version>
+                            <wildflyversion>${verifier.product.release.version}</wildflyversion>
```

Comment:
This is kind of a hijack of something meant and documented for a different purpose. If we want different uses we should probably rework the property and clarify it's use.

For the verifier use it's major.minor while wildflyversion in the past was just major. So there's a bit of a mismatch there.

Reply:
Any change to the properties should be discussed with Yeray though, as these are important for eap-ization and any changes will need to be accounted for there.

Reply:
Yeah. It's on my documentation todo list to change several constants to attributes.

Sure yeah. We don't have to use that version as it was set to just "13" before. I just figured the less places we need to make changes the better. I kind of wonder if these docs should just be in their own repository anyway.

Reply:
@jamezp I think a product.docs.server.version in the root pom next to verifier.product.release.version will make this cleaner

Reply:
Done.